app_legacy/Helpers/database/set-claim.php

```diff
@@ -15,39 +16,36 @@
  */
 function insertClaim(string $user, int $gameID, int $claimType, int $setType, int $special, int $permissions): bool
 {
-    sanitize_sql_inputs($user, $gameID, $claimType, $setType, $special);
-
-    if ($claimType == ClaimType::Primary) { // Primary Claim
+    if ($claimType === ClaimType::Primary) {
+        // Primary Claim
         // Prevent if user has no available slots except for when they are the sole dev of the set
-        if ($special == ClaimSpecial::None && getActiveClaimCount($user, false) >= permissionsToClaim($permissions)) {
+        if ($special === ClaimSpecial::None && getActiveClaimCount($user, false) >= permissionsToClaim($permissions)) {
             return false;
         }
 
-        $query = "
-            INSERT INTO
-                SetClaim (`User`, `GameID`, `ClaimType`, `SetType`, `Status`, `Extension`, `Special`, `Created`, `Finished` ,`Updated`)
-            VALUES
-                ('$user', '$gameID', '$claimType', '$setType', '" . ClaimStatus::Active . "', '0', '$special', NOW(), DATE_ADD(NOW(), INTERVAL 3 MONTH), NOW())";
+        $now = Carbon::now();
+        $finished = Carbon::now()->addMonths(3);
```

Comment:
Using Carbon for timestamps makes it mysql/sqlite agnostic; sqlite doesn't know `DATE_ADD(NOW(), INTERVAL 3 MONTH)`